src/wrapper/http.ts

```diff
@@ -64,21 +63,3 @@ export const serveRequestListener = (
   const server = http.createServer(requestListener);
   return startServer(server, port);
 };
-
-/**
```

Comment:
Once again you can't `spyOn` a function within the same file as something you are testing against